numpy/core/fromnumeric.py

```diff
@@ -438,7 +438,7 @@ def _repeat_dispatcher(a, repeats, axis=None):
 @array_function_dispatch(_repeat_dispatcher)
 def repeat(a, repeats, axis=None):
     """
-    Repeat elements of an array.
+    Repeat each element of an array after themselves
```

Comment:
@pierreloicq Thank you for sharing your thoughts on how to improve this part of the NumPy documentation! I'm not sure though if your suggestion makes the function description clearer.
What do yo think of this wording?

```suggestion
    Repeat each element of an array.
```

Reply:
Yes that makes sense too